package.json

```diff
@@ -1,82 +1,82 @@
 {
-    "name": "bramble",
```

Comment:
You've converted this file to 2-space indent from 4-space indent.  Can you change it back please?  It's important, since that's what Adobe does upstream, and if we want to merge with them in future, we need to not have git think we changed every line.